monarchmoney/monarchmoney.py

```diff
@@ -248,6 +248,112 @@ async def get_account_type_options(self) -> Dict[str, Any]:
             graphql_query=query,
         )
 
+    async def get_recent_account_balances(
+        self, start_date: Optional[date] = None
```

Comment:
For now, please change the `start_date` param and others in this PR to `str`.

Reply:
Changed this to a `str`